Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN tar -xzf "unity-version-manager-$UVM_VERSION.tar.gz" && rm "unity-version-ma
 
 RUN cd "unity-version-manager-$UVM_VERSION" && PATH="${HOME}/.cargo/bin:$PATH" make install
 
-FROM openjdk:8-jdk-buster
+FROM openjdk:11-jdk-buster
```

Comment:
We have to be careful with this. This for me means a breaking API change. I know that the rest of the java world handles this differently.

Reply:
The libraries are already running with java 11 for a good while now. I understand your worry and we can make this a breaking change, no problem. Alternatively we can keep java 8 by reimplementing the parts using the ProcessHandler lib to use the command line, for instance. But that goes against our idea of moving forward with java versions.

Reply:
I mean I know that our libraries run on java11. I still see this as a more general issue. Its a library. It is versioned by semver. The issue is that runtime dependencies are handled so and so and are never declared on the version level. So we can just update it but should make sure to mention this in the release notes.